djangocms_versioning/admin.py

```diff
@@ -753,6 +753,24 @@ def changelist_view(self, request, extra_context=None):
                 grouper=grouper,
                 title=_('Displaying versions of "{grouper}"').format(grouper=grouper),
             )
+            grouping_values = {
+                field: request.GET.get(field)
+                for field in versionable.extra_grouping_fields
+                if request.GET.get(field) is not None
+            }
+            grouping_values[versionable.grouper_field_name] = grouper
+            extra_context['latest_content'] = Version.objects.filter_by_grouping_values(
```

Comment:
Think this approach is fine, but this filtering is already done in ChangeList, is there no way to grab that?

Reply:
I'll look into it.